src/api/api.ts

```diff
@@ -1 +1,12 @@
 export const BASE_URL = 'https://mate.academy/students-api';
+
+export const request = (endpoint: string, method?:RequestInit) => {
```

Comment:
But better create 2 separe method like get and post
```suggestion
export const request = (endpoint: string, option?:RequestInit) => {
```